budcamp/Homework2/Homework2/IntWrapper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework2
+{
+    internal class IntWrapper
```

Comment:
you are not using this class anywhere, right? please remove it

Reply:
it was class to see how value and reference types works, yeap)